gcc/rust/lint/rust-lint-marklive.cc

```diff
@@ -113,12 +114,13 @@ MarkLive::visit (HIR::MethodCallExpr &expr)
 
   // node back to HIR
   HirId ref;
-  node_id_to_hir_id (expr.get_mappings ().get_crate_num (), ref_node_id, ref,
-		     expr.get_locus ());
+  rust_assert (
+    mappings->lookup_node_to_hir (expr.get_mappings ().get_crate_num (),
+				  ref_node_id, &ref));
```

Comment:
When assertion are disabled, `ref` will be uninit here

Reply:
Oops, I did't notice that.